api/src/main/java/io/opencensus/trace/EndSpanOptions.java

```diff
@@ -71,12 +70,12 @@ public static Builder builder() {
     /**
      * Sets the status for the {@link Span}.
      *
-     * <p>If set, this will override the default {@code Span} status. Default is {@link Status#OK}.
+     * <p>If set, this will override the status set via {@link Span#setStatus(Status)}.
      *
      * @param status the status.
      * @return this.
      */
-    public abstract Builder setStatus(Status status);
+    public abstract Builder setStatus(@Nullable Status status);
```

Comment:
Probably document why we want to set status to null here.

Reply:
Better to not allow it to be set to null.